configure.ac

```diff
@@ -137,6 +137,12 @@ AC_ARG_ENABLE([glibc-back-compat],
   [use_glibc_compat=$enableval],
   [use_glibc_compat=no])
 
+AC_ARG_ENABLE([zmq],
+  [AC_HELP_STRING([--disable-zmq],
```

Comment:
nit: still think we we should use `--enable-zmq` with a default `yes`. This would conform with `--enable-wallet`, etc.